website/docs/api/large-language-models.mdx

```diff
@@ -306,6 +300,87 @@ max_n_words = 20
 path = "summarization_examples.yml"
 ```
 
+### EL (Entity Linking) {id="nel"}
+
+The EL links recognized entities (see [NER](#ner)) against a provided knowledge
+base (KB). The EL task prompts the LLM to choose a shortlist of (the most
+likely) candidates from the KB. How this KB looks like and stores data can be
+arbitrary.
+
+Note that the documents run through the entity linking task are expected to have
+recognized entities in `.ents`. This can be achieved by - prior to running the
+EL task - running the [NER task](#ner), a trained spaCy NER model or by setting
+your docs' entities manually.
+
+In order to be able to pull data from the KB, an object implementing the
+`CandidateSelector` protocol has to be provided. This requires two functions:
+(1) `__call__()` to fetch candidate entities for entity mentions in the text
+(assumed to be available in `Doc.ents`) and (2) `get_entity_description()` to
+fetch descriptions for any given entity ID. Descriptions can be empty, but
+ideally provide more context for entities stored in the KB.
+
+`spacy-llm` provides a `CandidateSelector` implementation
+(`spacy.CandidateSelector.v1`) that leverages a spaCy pipeline with an
+`entity_linking` component to select candidates. Note that this pipeline doesn't
+have to provide a trained EL model, merely its default (or custom) candidate
+selection capabilities.
+
+#### spacy.EL.v1 {id="el-v1"}
```

Comment:
Isn't this actually called `spacy.EntityLinker.v1`  in the PR? We need to make sure we're consistent throughout (I also found a reference to "EntityLinking" in the other PR)

Reply:
Ugh, good catch. Fixed in https://github.com/explosion/spaCy/pull/12988/commits/fa3645d3876cf83ce9205cd2542b06270d40763e.